### PR TITLE
Restore API for perform_align function

### PR DESCRIPTION
**Pull request by stsci-hack**
The interface to perform_align() could not return any values when wrapped by the util.with_logging decorator.  However, the pipeline relies on the returned table to determine what to do next.  This set of changes restores the interface so that perform_align( ) once again returns the filteredTable.

### CODE REVIEW

File: drizzlepac/runastrodriz.py [hunk 1]
```diff
@@ -72,6 +72,8 @@
 
 import math
 
+import math
+
```
Comment (mdlpstsci) [inline drizzlepac/runastrodriz.py]: Why is this import here twice?

Comment (stsci-hack) [reply drizzlepac/runastrodriz.py]: Actually, it was just one of a few unused import statements which I removed.
